core/src/main/java/com/linecorp/armeria/common/logging/ContentPreviewerFactoryBuilder.java

```diff
@@ -199,17 +240,17 @@ public ContentPreviewerFactory build() {
     }
 
     private static BiPredicate<? super RequestContext, ? super HttpHeaders> previewerPredicate(
-            MediaTypeSet mediaTypeSet) {
-        requireNonNull(mediaTypeSet, "mediaTypesSet");
+            Iterable<MediaType> meidaTypes) {
         return (ctx, headers) -> {
             final MediaType contentType = headers.contentType();
             if (contentType == null) {
                 return false;
             }
-            return mediaTypeSet.match(contentType) != null;
+            return Streams.stream(meidaTypes).anyMatch(contentType::belongsTo);
         };
     }
```

Comment:
How about keep accepting `MediaTypeSet`? The callers of this method could create a `MediaTypeSet` from the given `MediaType`s.